README.md

```diff
@@ -40,5 +45,19 @@
   > bundle:start <net.staticsnow:nexus-repository-apt ID>
   ```
 
+#### Permanent Install
+
+For more permanent installs of the nexus-repository-apt plugin, you will need to have the Repository Manager stopped
```

Comment:
u don't actually need to stop it - the deploy directory is monitored while Nexus is running and the bundle will deploy within 60 seconds of being put there.